install.sh

```diff
@@ -187,14 +186,27 @@ check_deps() {
 node(){
   # clean up 
   rm -rf node 
+  rm -f SHASUMS256.txt.asc
   rm -rf node-$NODE_VERSION*
-  
+
   echo :Installing Node $NODE_VERSION
-  
+
+  gpg --list-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D 2>&1 >/dev/null || gpg --keyserver pool.sks-keyservers.net --recv-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D
```

Comment:
Thanks for the explanation! So in this case gpg checking won't make whole thing more secure since we download many other dependencies via https, but there is no harm in doing it if it doesn't break installation falsely.

I've cherrypicked virtualenv fix and changed node url to https in https://github.com/c9/install/pull/41.

For gpg checking we probably can add something like
```sh
verify_node() {
  if has gpg; then
    rm -f SHASUMS256.txt.asc
    gpg --list-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D 2>&1 >/dev/null || gpg --keyserver pool.sks-keyservers.net --recv-keys 7937DFD2AB06298B2293C3187D33FF9D0246406D
    gpg --list-keys 114F43EE0176B71C7BC219DD50A3051F888C628D 2>&1 >/dev/null || gpg --keyserver pool.sks-keyservers.net --recv-keys 114F43EE0176B71C7BC219DD50A3051F888C628D
    $DOWNLOAD "http://nodejs.org/dist/$NODE_VERSION/SHASUMS256.txt.asc"
    gpg --verify SHASUMS256.txt.asc && grep " node-$NODE_VERSION-$1-$2.tar.gz\$" SHASUMS256.txt.asc | sha256sum -c - || exit 1
  fi
}
```
with some explanation of the error.
https://github.com/c9/install/pull/38#